### PR TITLE
add endpoint to scanner and workServer, fix case where matches.player…

### PR DESCRIPTION
**Pull request by albertcui**
…s isn't there?

### CODE REVIEW

File: scanner.js [hunk 1]
```diff
@@ -15,6 +15,29 @@ var constants = require('./constants');
...
+var app = express();
+var port = config.PORT || config.SCANNER_PORT;
+var startedAt = moment();
+
+app.route("/").get(function(res, req) {
+    redis.get("match_seq_num", function(err, result) {
+        res.json({
+            started_at: startedAt.format(),
+            started_ago: moment.fromNow(),
+            match_seq_num: result || "NOT FOUND"
```
Comment (howardchung) [inline scanner.js]: scanner supports several modes.  Also won't this not update with the scanner as it scans?

**Comment by howardchung** [comment]
Why do we need this :P

**Comment by albertcui** [comment]
Because this way we know if one of our services are down

**Comment by howardchung** [comment]
How are you going to monitor?  If they stall, then uptime check won't alert because the endpoint will still be up

**Comment by albertcui** [comment]
The stalls are caused by crashes for the most part.

**Comment by howardchung** [comment]
But pm2 will just bring them back up

**Comment by howardchung** [comment]
If they are stalling it probably means we're not calling back somewhere

**Comment by albertcui** [comment]
Doesn't ways happen. Today's work server crashes were not brought back.

**Comment by howardchung** [comment]
If it's constantly crashing and coming back an uptime check may not catch
it.

**Comment by albertcui** [comment]
Yes, but I'm saying today it crashed and never came up. I know because I
manually hit the endpoint.

**Comment by howardchung** [comment]
You deployed this to production?

**Comment by albertcui** [comment]
No I'm saying I manually hit the parse endpoint on work server.

**Pull request merged**